internal/v1/handler_blueprints_test.go

```diff
@@ -185,7 +185,7 @@ func TestHandlers_ComposeBlueprint(t *testing.T) {
 	err = dbase.InsertBlueprint(ctx, id, versionId, "000000", "000000", name, description, message)
 	require.NoError(t, err)
 
-	respStatusCode, body := tutils.PostResponseBody(t, fmt.Sprintf("http://localhost:8086/api/image-builder/v1/experimental/blueprints/%s/compose", id.String()), map[string]string{})
+	respStatusCode, body := tutils.PostResponseBody(t, fmt.Sprintf("http://localhost:8086/api/image-builder/v1/experimental/blueprints/%s/compose", id.String()), ComposeBlueprintJSONBody{})
```

Comment:
Please change this test case to send zero body, this here actually sends the following body:

`{}`

This is not what UI does today, I want to be sure we will not break it when we merge this.

In case you are wondering why there was a map previously, somebody was lazy and since the helper function requires `any` value (must not be `nil`), they used a map for some reason:

```
func PostResponseBody(t *testing.T, url string, compose interface{}) (int, string)
```

The proper solution is/was to modify the helper `PostResponseBody` to handle `nil` correctly: send no body.

Reply:
that exactelly what I thoght, will do it, thank you!

Reply:
Yeah, these [lazy people](https://github.com/osbuild/image-builder/commit/3c932f7e4da41c7c16d8b68594cf9e15b65c941c) :roll_eyes: (to my defense I probably have copied it somewhere).
If you struggle to actually send empty request, let me know, I can help out correct my past mistakes :)

Reply:
Wow I was pretty sure it was me, okay. :-D